exercises/pangram/stack.yaml

```diff
@@ -0,0 +1 @@
+resolver: lts-8.6
```

Comment:
good time to ask now about whether there is a policy:

* Do we want to keep the same version across the same track?
* Or is it acceptable to let each exercise have its own version?